### ISSUE TITLE
annotations

### ISSUE THREAD

**Issue by pandichef** [ivelum/djangoql#49]
Awesome project!  

Is there a way to put a "not" in front of a long complex query?  

Also, it would be great if support were added for annotations i.e., calculated columns.  (Note: I'm using the DjangoQLQuerySet.as_manager() version.)

**Comment by stebunovd** [comment]
> Is there a way to put a "not" in front of a long complex query?

no, it's not supported at the moment. However, any query can be rewritten in a way that doesn't require "not".
